Jenkinsfile.nightly-build

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright © 2018 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+import java.text.SimpleDateFormat
+
+properties([pipelineTriggers([cron('15 2 * * *')])])
+
+def dateFormat = new SimpleDateFormat("yyyyMMdd")
+def date = new Date()
+def timestamp = dateFormat.format(date)
+
+def generateSrcCodeRelease() {
+		sh """
+		npm install
+		node ./node_modules/.bin/grunt release
+		jq -r '.version' config.json > "/tmp/version"
+		"""
+	return readFile("/tmp/version").trim()
+}
+
+pipeline {
+	agent { 
+		node { 
+			label 'node-07'
+			customWorkspace "workspace/${URLDecoder.decode(JOB_NAME)}/${BUILD_NUMBER}"
+			}
+	}
+	stages {
+		stage('Generate release') {
+			steps {
```

Comment:
We should have the build steps (the whole `sh` step) here to make things clearer.